build/yaml/webhook/manifests.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+  annotations:
+    cert-manager.io/inject-ca-from: vmware-system-nsx/nsx-operator-webhook-cert
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: subnetset
+      namespace: vmware-system-nsx
+      # kubebuilder webhookpath.
+      path: /validate-nsx-vmware-com-v1alpha1-subnetset
+  failurePolicy: Fail
+  name: vsubnetset.kb.io
+  rules:
```

Comment:
could we add the below objectSelector to only select the default subnetset for Pod and VM? for other Subnetsets, no need to validate using this webhook.
objectSelector:
    matchExpressions:
      key: nsxoperator.vmware.com/default-subnetset-for
      operator: In
      values: ["Pod", "VirtualMachine"]

Reply:
Good point, done.